payments/management/commands/reauthorize.py

```diff
@@ -0,0 +1,42 @@
+from django.core.management.base import BaseCommand
+from django.conf import settings
+from datetime import timedelta
+from django.utils import timezone
+
+import stripe
+
+from auctions.models import Offer
+from ... import utils as payments
+
+stripe.api_key = settings.STRIPE_SECRET_KEY
+
+
+class Command(BaseCommand):
+
+    def add_arguments(self, parser):
+        parser.add_argument('day_limit', nargs='+', type=int)
+
+    def handle(self, *args, **options):
+        day_limit = options['day_limit'][0]
+
```

Comment:
let's add a `#TODO` above this to move the expiration date calculation into the `filter` to only return offers that will expire.